src/stores/clusterapps/actions.ts

```diff
@@ -0,0 +1,65 @@
+import GiantSwarm, { V4App } from 'giantswarm';
+import { FlashMessage, messageTTL, messageType } from 'lib/flashMessage';
+
+import { createAsynchronousAction } from '../asynchronousAction';
+
+interface IRequest {
+  clusterId: string;
+}
+
+interface IResponse {
+  apps: V4App[];
+  clusterId: string;
+}
+/**
+ * Loads apps for a cluster.
+ *
+ * @param {String} clusterId Cluster ID
+ */
+export const loadClusterApps = createAsynchronousAction<
+  IRequest,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  any,
```

Comment:
It might be a monumental undertaking, but if we get a full interface for the State, then we can get rid of this any. I might try it out tonight.